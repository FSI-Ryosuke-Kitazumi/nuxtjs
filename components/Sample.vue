<script setup lang="ts">
const props = defineProps<{
  title: string;
}>();

console.log(props.title);
</script>

<template>
  <div>
    <div>sample</div>
    <slot />
    <p>{{ title }}</p>
  </div>
</template>
