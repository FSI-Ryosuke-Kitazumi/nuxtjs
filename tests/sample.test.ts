import { expect, test } from "vitest";
import Sample from "../components/Sample.vue";
import { mount } from "@vue/test-utils";

test("my test", () => {
  const component = mount(Sample);
  expect(component.html()).toMatchSnapshot();
});
