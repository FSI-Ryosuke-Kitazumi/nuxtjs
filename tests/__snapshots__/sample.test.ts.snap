// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`my test 1`] = `"<div>sample</div>"`;
